src/tests/all.mk

```diff
@@ -20,7 +20,8 @@ raddb/test.conf:
 #
 # Only redirect STDOUT, which should contain details of why the test failed.
 # Don't molest STDERR as this may be used to receive output from a debugger.
-radiusd-c $(BUILD_DIR)/tests/radiusd-c: raddb/test.conf ${BUILD_DIR}/bin/radiusd $(GENERATED_CERT_FILES) | $(BUILD_DIR)/tests build.raddb
+.PHONY:
```

Comment:
I don't think an empty `.PHONY:` does anything.  And the rule `$(BUILD_DIR)/tests/radiusd-c` creates an actual file, so it shouldn't be `.PHONY:`